### DISCUSSION TITLE
Sparkle 2.5

### DISCUSSION THREAD

**Discussion by zorgiepoo** [sparkle-project/Sparkle#2411] [Announcements]
Sparkle 2.5 has been [released](https://github.com/sparkle-project/Sparkle/releases).

It includes enhancements to the release notes view such as:
* Adapting release notes based on the currently installed version with [sample documentation](https://sparkle-project.org/documentation/publishing#adapting-release-notes-based-on-currently-installed-version). This allows for de-empasizing or hiding past already installed releases for example.
* Allowing custom URL schemes to be clicked in Sparkle's release notes view. By default, custom URL schemes are rejected for safety purposes. This can now be customized via the `SUAllowedURLSchemes` key noted in [Customizations.](https://sparkle-project.org/documentation/customization/)

It also adapts macOS Sonoma's cooperative app activation APIs (rather than using deprecated APIs), which should be a mostly invisible change, and better pre-warms installs before needing to relaunch.

**Comment by ychin** [comment]
Hi! I like the idea of being able to show different release notes based on the current version the user has installed. However, the design doesn't really make sense to me, and I posted on https://github.com/sparkle-project/Sparkle/pull/2373#issuecomment-1681294359 for more elaboration.
